docs/design/field-references.md

```diff
@@ -57,3 +57,65 @@ spec:
           name: sleeper3
         spec: "{{{sleeper2#spec}}}"
 ```
+
+## Referring to ServiceBinding outputs
+
+When Service Catalog processes a ServiceBinding, the output is placed in a secret
+(since they might be secret). If they're not secret, it's convenient to directly
+reference them in the bundle. This can be done by using `dependency:bindsecret#secretkey`.
```

Comment:
Yeah, I was in two minds. I ended up deciding 'bindsecret' implied its own special semantics, so it may as well have enough magic to make the operation as clear as possible (thinking we could add a new magic string if we ever decided we wanted the secret). But, will change.